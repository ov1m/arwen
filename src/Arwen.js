import React, { Component } from 'react';
import { Text } from 'react-native';

class Arwen extends Component {
  render() {
    return (
			<Text>Open up main.js to start working on your app! </Text>
    );
  }
}

export default Arwen;