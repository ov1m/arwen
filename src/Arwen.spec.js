import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Arwen from './Arwen';

test('renders correctly', () => {
	const tree = renderer.create(
		<Arwen />
	).toJSON();
	expect(tree).toMatchSnapshot();
});
