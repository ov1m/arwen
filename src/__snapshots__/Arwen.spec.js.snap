// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  Open up main.js to start working on your app! 
</Text>
`;
